src/Aspire.Hosting.SurrealDb/Aspire.Hosting.SurrealDb.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>aspire integration hosting surrealdb</PackageTags>
+    <Description>SurrealDB support for .NET Aspire.</Description>
+    <PackageIconFullPath>$(SharedDir)surrealdb.png</PackageIconFullPath>
```

Comment:
Do we have permission to use surrealdb icon?

Reply:
Yes, we do.